dashboard/src/components/AppView/AppView.tsx

```diff
@@ -259,8 +274,17 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
     }
   }
 
-  private deploymentArray(): IResource[] {
-    return Object.keys(this.state.deployments).map(k => this.state.deployments[k]);
+  // Retrieve the deployments/service/ingresses if they are already loaded
+  private arrayFromState(type: string): Array<IKubeItem<any>> {
```

Comment:
isn't it going to be an `IKubeItem<IDeployment | IService | IIngress>` or if we need to be more general `IKubeItem<IResource>`. We should try to avoid using `any` if possible.

Reply:
_removed this method_